src/page/homepage/Homepage.jsx

```diff
@@ -1,35 +1,53 @@
 import React, { useEffect, useState } from 'react';
-import axios from 'axios';
 import {
   AddNoteBtn,
+  Filter,
   NoteCard,
   NotesInputModal,
 } from '../../component/component-index';
-import { useToken, useNotesList } from '../../context/context-index';
+import { useToken, useNotesList, useFilter } from '../../context/context-index';
+import { getTagData, getSortedData } from '../../utilities/utilities-index';
 import './Homepage.css';
+import { getNote } from '../../api-call/api-index';
 
 export const HomePage = () => {
   const { notesList, setNotesList } = useNotesList();
+  const { tagState, sortByState } = useFilter();
   const { token } = useToken();
   useEffect(() => {
     (async () => {
-      const response = await axios.get('/api/notes', {
-        headers: { authorization: token },
-      });
+      const response = await getNote(token);
       if (response.status === 200 || response.status === 201) {
         setNotesList([...response.data.notes]);
       }
```

Comment:
Add error handling in the else part

Reply:
Will work on error handling thanks for the suggestions.